deliveries/skyekiwi-protocol-milestone-2.md

```diff
@@ -0,0 +1,26 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/skyekiwi-protocol.md
+* **Milestone Number:** 1
```

Comment:
I believe you're submitting M2, correct?

Reply:
Indeed!😂